### PR TITLE
Updated votingDeadline variable to minExecutionDate

### PR DESCRIPTION
**Pull request by ShubhamTatvamasi**

### CODE REVIEW

File: solidity/dao-congress.sol [hunk 1]
```diff
@@ -253,8 +253,9 @@ contract Congress is owned, tokenRecipient {
         onlyMembers public
         returns (uint voteID)
     {
-        Proposal storage p = proposals[proposalNumber];         // Get the proposal
-        require(!p.voted[msg.sender]);         // If has already voted, cancel
+        Proposal storage p = proposals[proposalNumber]; // Get the proposal
+        require(now < p.minExecutionDate);              // If time has left before voting deadline
```
Comment (alexvandesande) [inline solidity/dao-congress.sol]: You can remove this line

**Comment by alexvandesande** [comment]
Thanks for your contribution

File: solidity/dao-congress.sol (hunk 1 above)
Comment (ShubhamTatvamasi) [reply solidity/dao-congress.sol]: I added this line 257 only so people don't execute vote function after the voting time is over.

Comment (ShubhamTatvamasi) [reply solidity/dao-congress.sol]: I have removed the line 257 in new commit, please see it.

**Comment by alexvandesande** [comment]
Merged, thanks!

**Pull request merged**